bin/bios-baseline.sh

```diff
@@ -220,12 +220,12 @@ function run_ilo() {
             ilorest --nologo login ${ncn_bmc} -u ${bmc_username} -p ${bmc_password} >/dev/null
         fi
 
-        ilo_config 2>&1 >$LOG_DIR/${ncn_bmc}.log
-        ilorest --nologo logout 2>&1 >/dev/null
+        ilo_config >$LOG_DIR/${ncn_bmc}.log 2>&12>&1
```

Comment:
`&12` you say?

Reply:
Oops.  I'll fix that up

Reply:
ok I meant to just to the shellcheck disable for these.  I committed my attempted changes, but everything is fixed up now to just disable these warnings for now